test/e2e/e2e_test.go

```diff
@@ -49,22 +50,27 @@ func getConfig() E2EConfig {
 
 var _ = Describe("The Orbs Network", func() {
 	It("accepts a transaction and reflects the state change after it is committed", func(done Done) {
-		var node bootstrap.Node
+		var nodes []bootstrap.Node
 
 		// TODO: kill me - why do we need this override?
 		if getConfig().Bootstrap {
 			gossipTransport := gossipAdapter.NewTamperingTransport()
-			nodeKeyPair := keys.Ed25519KeyPairForTests(0)
-			node = bootstrap.NewNode(
-				":8080",
-				nodeKeyPair.PublicKey(),
-				nodeKeyPair.PrivateKey(),
-				map[string]config.FederationNode{nodeKeyPair.PublicKey().KeyForMap(): config.NewHardCodedFederationNode(nodeKeyPair.PublicKey())},
-				nodeKeyPair.PublicKey(), // we are the leader
-				consensus.CONSENSUS_ALGO_TYPE_LEAN_HELIX,
-				2*1000,
-				gossipTransport,
-			)
+
+			for i := 0; i < 3; i++ {
```

Comment:
this is confusing and not scalable. Someone adding a 4th node to the compose file will not think that they need to change this number here. Either pass NUM_OF_NODES from env or, even better, deduct it somehow.

Reply:
I think this should be addressed in a separate pr.